.github/workflows/ci.yml

```diff
@@ -59,8 +59,8 @@ jobs:
           - make integration-test
           - TARGET=dev .github/smoke-test.sh
           - TARGET=prod .github/smoke-test.sh
-          - .github/analyse.sh
-          - .github/hypertest.sh
+          - TARGET=prod make api-validate
+          - TARGET=prod make api-test
         include:
           - target: prod
             test: TARGET=prod .github/smoke-test.sh
```

Comment:
Unrelated, but not sure why this duplicate the above?

Reply:
It is not a duplicate, but an override of that item; modifies `{{ matrix.target }}` in https://github.com/libero/article-store/pull/272/files#diff-e9f950f17198d3d5e3122a44230a09b9R76 to run on the production image rather than the default development one.

Reply:
Ah, is different to Travis. Adds config rather than new jobs (https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-configurations-in-a-matrix-build).